apps/website/src/content/docs/informationpanel.mdx

```diff
@@ -1,22 +1,65 @@
 ---
 title: Information panel
 description: A way to view additional attributes without leaving the list of items.
-propsPath: '@itwin/itwinui-react/esm/core/InformationPanel/InformationPanel.d.ts'
 thumbnail: InformationPanel
 ---
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='informationpanel' />
-
 <LiveExample src='InformationPanel.main.tsx'>
   <AllExamples.InformationPanelMainExample client:load />
 </LiveExample>
 
-The information panel is a pattern used primarily in tables. It allows viewing or editing additional details about selected item(s) without leaving the table.
+The Information Panel is designed for complex tables, offering a user-friendly way to explore content without overwhelming details or horizontal scrolling. It enhances data presentation in tables when needed.
+
+## Usage
+
+The Information Panel component is made up of customizable subcomponents. Here’s what the basic usage looks like:
+
+```tsx
+<InformationPanelWrapper>
+  <Table ... /> {/* component to overlay */}
+
+  <InformationPanel isOpen={true}>
+    <InformationPanelHeader onClose={() => {}}>
+      <Text variant='subheading'>InfoPanel heading</Text>
+    </InformationPanelHeader>
+
+    <InformationPanelBody>
+      <Text>Metadata</Text>
+      <InformationPanelContent ... />
+      <Text>Attributes</Text>
+      <InformationPanelContent ... />
+    </InformationPanelBody>
+
+  </InformationPanel>
+</InformationPanelWrapper>
+```
+
+### Overlay Component
```

Comment:
I don't think this content should be under a new sub-heading. Because this seems like the continuation of "Usage" itself

Reply:
i've rewritten the entire Usage section to flow more naturally. let me know what you think

Reply:
The changes are great! Concise yet descriptive. I left a few comments below. So I'm resolving this comment.